app/services/default_values_object.rb

```diff
@@ -0,0 +1,21 @@
+class DefaultValuesObject
```

Comment:
Ah I see. So instead what we could have done is look at the methods on the form to see if there are setters for its attributes. I think I'm going to do more research on feature-envy so next time we come across this, It'll be easier to spot and mitigate.